tests/drivers/spi/spi_loopback/boards/nucleo_l552ze_q.overlay

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright (c) 2021 STMicroelectronics
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+&spi1 {
```

Comment:
Why adding spi1 in the overlay of the test ?
Why not adding spi1 in board Devicetree ?

Reply:
so that the dts is changed just for _that_ test,

Reply:
Please move spi1 definition (pins and status) to board description.
(dma part should stay here)

Reply:
Request still stands